convert2rhel/main.py

```diff
@@ -145,6 +145,10 @@ def pre_ponr_conversion():
     loggerinst.task("Convert: Remove blacklisted packages")
     pkghandler.remove_blacklisted_pkgs()
 
+    # checking RHN Classic
+    loggerinst.task("Convert: Checking RHN Classic")
+    pkghandler.check_rhn_classic()
```

Comment:
Since this just checks if the system is on RHN Classic it doesn't make much sense to have this in `pre_ponr_conversion` since it's not doing any tasks. I'd suggest we move this just under `rhelvariant.determine_rhel_variant()`